CITATION.md

```diff
@@ -0,0 +1,43 @@
+
+# Citing Mathlib
+
+You can cite `mathlib` using the following BibTeX entry:
+
+```bib
+@InProceedings{   mathlib2020,
+  author        = {The mathlib Community},
+  title         = {The Lean Mathematical Library},
+  year          = {2020},
+  isbn          = {9781450370974},
+  publisher     = {Association for Computing Machinery},
+  address       = {New York, NY, USA},
+  url           = {https://doi.org/10.1145/3372885.3373824},
+  doi           = {10.1145/3372885.3373824},
+  booktitle     = {Proceedings of the 9th ACM SIGPLAN International
+                  Conference on Certified Programs and Proofs},
+  pages         = {367-381},
+  numpages      = {15},
+  keywords      = {formal proof, formal library, Lean, mathlib},
+  location      = {New Orleans, LA, USA},
+  series        = {CPP 2020}
+}
+```
+
+The `mathlib` project is actively maintained and updated. In the interest of reproducibility, you may want to provide a commit hash referencing the version of `mathlib` your project uses alongside your citation.
```

Comment:
Anyone who cares about reproducibility should start by publishing their own lean code. If they do that via the documented method they will end up with the git sha in their lean project.toml.

So I think it would be more productive to encourage people to use `leanproject` properly and archive their code appropriately rather than put a useless sha1 in a pdf.

Reply:
Ok, updated to:

The `mathlib` project is actively maintained and updated. In the interest of reproducibility, you may want to manage your code's dependency on `mathlib` using the [`leanproject`](https://leanprover-community.github.io/leanproject.html) command-line tool. This will help ensure others can retrieve the right version of `mathlib` for your project.